client/driver/rkt.go

```diff
@@ -151,8 +151,11 @@ func (d *RktDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool, e
 	rktMatches := reRktVersion.FindStringSubmatch(out)
 	appcMatches := reAppcVersion.FindStringSubmatch(out)
 	if len(rktMatches) != 2 || len(appcMatches) != 2 {
+		if currentlyEnabled {
```

Comment:
Or I am wondering that this entire Fingerprint can be skipped, when `currentlyEnabled (node.Attributes[rktDriverAttr])` is false.